qt/aqt/sound.py

```diff
@@ -671,7 +671,7 @@ def _start_recording(self) -> None:
                 namedtmp("rec.wav"), self.mw, self._parent
             )
         else:
-            from .qt5 import QtAudioInputRecorder as Qt5Recorder
+            from aqt.qt.qt5extra import QtAudioInputRecorder as Qt5Recorder
```

Comment:
Since this is a package now, maybe aqt.qt.qt5_audio or similar would be a better choice?